### DISCUSSION TITLE
Croatian translation

### DISCUSSION THREAD

**Discussion by abajo** [rustdesk/rustdesk#7635] [General]
All lines translated:
https://pastebin.com/DPtuhbMj

**Comment by rustdesk** [comment]
Thanks, good job, please submit a PR.

**Comment by abajo** [comment]
In version 1.2.4. when selecting the language in Croatian, it switches to Hebrew.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/commit/ead8a48436bbb26d97c1fadca04ccb1b2f681a58
